concurrent.el

```diff
@@ -1,11 +1,11 @@
-;;; concurrent.el --- Concurrent utility functions for emacs lisp
+;;; concurrent.el --- Concurrent utility functions for emacs lisp  -*- lexical-binding: t; -*-
 
 ;; Copyright (C) 2010-2016  SAKURAI Masashi
 
 ;; Author: SAKURAI Masashi <m.sakurai at kiwanami.net>
 ;; Version: 0.4.0
 ;; Keywords: deferred, async, concurrent
-;; Package-Requires: ((deferred "0.4.0"))
+;; Package-Requires: ((emacs "24.3") (cl-lib "1.0") (deferred "0.4.0"))
```

Comment:
You need not to specify `cl-lib` because `cl-lib` was bundled since Emacs 24.3. And `cl-lib` 1.0 is not provided as package.